code/client/src/components/LocalExport.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import protobuf from 'protobufjs/light'
+import message from '../message'
+import storage from '../storage'
+import { Button } from 'antd'
+import { ExportOutlined, LoadingOutlined } from '@ant-design/icons'
+import { useState } from 'react'
+
+const Field = protobuf.Field
+
+export function LocalExportMessage (properties) {
```

Comment:
I wonder if there are more syntactically cleaner / modern ways of defining it.

Reply:
I will look into it. It has multiple ways to define the message.

Reply:
Although there are multiple ways to declare the proto type, none of it is cleaner than the current option. The only cleaner/modern option is using decorator with TypeScript.

https://github.com/protobufjs/protobuf.js#examples